packages/graphback-core/src/db/databaseMapper.ts

```diff
@@ -0,0 +1,56 @@
+import { GraphQLField, GraphQLFieldMap, GraphQLObjectType, GraphQLSchema } from "graphql";
+import { parseDbAnnotations } from '../annotations/parser';
+import { getUserModels } from '../crud';
+import { defaultTableNameTransform } from './defaultNameTransforms';
+import { getPrimaryKey } from './getPrimaryKey';
+
+export interface DatabaseMap {
+  modelMap: ModelMap[]
+}
+
+export interface ModelMap {
+  typeName: string
+  tableName: string
+  id: string
+  fieldMap: any
+}
+
+export const getModelTableMappings = (schema: GraphQLSchema): DatabaseMap => {
```

Comment:
Can we use array of model types here. It will help us to make this clearly about the CRUD types. If someone do not map element as model it should not be included (equivalent to db.disable

Reply:
This is kinda complex topic so happy to have chat about this.

Reply:
Yeah I had this changed already but not pushed!